express/blocks/how-to-steps/how-to-steps.css

```diff
@@ -238,3 +239,14 @@ main .section .template-x-heading {
     margin: 0 auto;
   }
 }
+
+@media (max-width: 900px) {
+  main .section:has(.content:first-child + .how-to-steps) .content{
```

Comment:
you shouldn't mix min-width & max-width in a css file unless you want css to apply to a very specific range :) 
In general as well, I prefer using min-width most of the time, because that's a more mobile-first approach. 

here, the width should be max 810px btw, not fit-content.